apps/web/src/LandingPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from 'react';
+
+function LandingPage() {
+    useEffect(() => {
+        const handleMouseClick = () => {
+            handlePageTransition();
+        };
+        const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
Although it's not strictly necessary, for arrow functions in TypeScript, annotate the return type.